keel-core/src/main/kotlin/com/netflix/spinnaker/keel/persistence/memory/InMemoryUnhappyVetoRepository.kt

```diff
@@ -28,42 +28,45 @@ class InMemoryUnhappyVetoRepository(
 
   private val resources: MutableMap<String, Record> = mutableMapOf()
 
-  override fun markUnhappyForWaitingTime(resourceId: String, application: String, wait: Duration) {
+  override fun markUnhappyForWaitingTime(resourceId: String, application: String, wait: Duration?) {
```

Comment:
As an aside, how close are we to eliminating the in memory repositories altogether?

Reply:
I've got most of the usages removed. I'll continue with that work this week.